src/apig_wsgi.py

```diff
@@ -38,6 +39,7 @@ def handler(event, context):
         response = Response(
             binary_support=binary_support,
             non_binary_content_type_prefixes=non_binary_content_type_prefixes,
+            multi_value_header_support=environ["apig_wsgi.multivalue_headers"],
```

Comment:
I've normalized both the names `multi_value_header_support` and `multivalue_headers` to `multi_value_headers`, since that's the snake case version of the JSON name